mission/src/views/ItemInfo.vue

```diff
@@ -1,24 +1,269 @@
+/* eslint-disable vue/require-v-for-key */
 <template>
-<div id='item-info-page'></div>
+  <div id="item-info-page">
+    <div class="product-image">
+      <img class="product" src="@/assets/images/dotShirt.jpg" />
+    </div>
+    <div class="product-detail">
+      <div class="product-brand">
+        <img class="brand-image" src="@/assets/images/brand.jpg" />
+        <div class="brand-info">
+          <div>{{ productBrand }}</div>
+          <div class="brand-tag">
+            <!-- eslint-disable-next-line vue/require-v-for-key -->
+            <div class="brand-text" v-for="tag in tags">{{ tag }}</div>
+          </div>
+        </div>
+        <div v-if="!isLove">
+          <img class="star-button" v-on:click="click" src="@/assets/images/star.svg" />
+        </div>
+        <div v-else>
+          <img class="star-button" v-on:click="click" src="@/assets/images/starFill.svg" />
+        </div>
+      </div>
+
+      <div class="product-info">
+        <div class="product-title">{{ productTitle }}</div>
+        <div class="product-discount">
+          <div class="product-onSale" v-if="onSale">
+            <div class="discount-percent">{{ discountPricePercent }}</div>
+            <div class="discount-price">{{ discountPrice }}</div>
+          </div>
+          <div class="original-price">{{ originalPrice }}</div>
+        </div>
+        <h2>상품 정보</h2>
+        <h3>{{ productContent }}</h3>
+        <img src="@/assets/images/slacks.jpg" />
+        <p>{{ productSubContent }}</p>
+      </div>
+      <!-- 컴포넌트화 -->
+      <div class="product-review" v-if="isReview">
+        <h2>리뷰</h2>
+        <div class="review-box">
+          <div class="review-info">
+            <div class="review-id">{{ reviewId }}</div>
+            <div class="review-date">{{ reviewDate }}</div>
+          </div>
+          <div class="review-container">
+            <div class="review-content">
+              <p>{{ reviewTag[0] }}</p>
+              <div class="reviewer-content">{{ reviewContent }}</div>
+            </div>
+            <div class="reviewer-image">
+              <img class="review-img" src="@/assets/images/good.jpg" />
+            </div>
+          </div>
+        </div>
+        <footer class="buy-container">
+          <button class="buy-button">{{ `${discountPrice} 구매}` }}</button>
+        </footer>
+      </div>
+    </div>
+    <ProductDetail />
+    <Review />
+  </div>
 </template>
 
 <script>
+import ProductDetail from '@/components/ProductDetail.vue';
+import Review from '@/components/Review.vue';
+
 export default {
   name: 'ItemInfoPage',
+  component: {
+    ProductDetail,
+    Review,
+  },
   data() {
     return {
-
+      product: 'clothes',
+      productBrand: 'AND MORE',
+      tags: ['#남성', '#셔츠', '#땡땡이'],
```

Comment:
```suggestion
      tags: ['남성', '셔츠', '땡땡이'],
```
태그 앞에 붙는 `#`은 공통되니 html 코드에서 처리해도 좋을 것 같아요.  :)